### PR TITLE
docs: misc fixes

### PR DESCRIPTION
**Pull request by anshbansal**
## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ Running `datahub init` will allow you to customize the datahub instance you are
 
 **_Note_**: Provide your GMS instance's host when the prompt asks you for the DataHub host.
 
-#### Environment variables supported
+### Environment variables supported
```
Comment (anshbansal) [inline docs/cli.md:89]: This should appear in side bar now

File: docs/debugging.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Debugging Guide
 
+## Logo for my platform is not appearing on the Home Page or search results
```
Comment (anshbansal) [inline docs/debugging.md:3]: Logo missing is not tied to custom data platform anywhere in the docs. This should solve that

Comment (hsheth2) [inline docs/debugging.md:3]: this shouldn't be the very first item - the quickstart debugging docs are much more important

**Review by hsheth2 — APPROVED** [review]
one change but lgtm other than that

File: docs/debugging.md (hunk 2 above)
Comment (anshbansal) [reply docs/debugging.md]: Did not realise this was only for quickstart debugging. Changed the name, moved that FAQ to it's own section.

Comment (hsheth2) [reply docs/debugging.md]: It doesn't need to be exclusive to quickstart - my comment was that this new section should be at the bottom of the doc

**Pull request merged**